### ISSUE TITLE
The single body parameter example in the documentation does not work.

### ISSUE THREAD

**Issue by ugurpy** [tiangolo/fastapi#4164]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [ ] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from typing import Optional

from fastapi import Body, FastAPI
from pydantic import BaseModel

app = FastAPI()


class Item(BaseModel):
    name: str
    description: Optional[str] = None
    price: float
    tax: Optional[float] = None


@app.put("/items/{item_id}")
async def update_item(item_id: int, item: Item = Body(..., embed=True)):
    results = {"item_id": item_id, "item": item}
    return results
```

### Description

The example described under the heading "Embed a single body parameter" in the "Body - Multiple Parameters" section of the user manual does not work as stated.

postman put request:
![Screenshot from 2021-11-09 19-14-29](https://user-images.githubusercontent.com/26383184/140962011-77ad028d-0180-41fe-a0c9-4a7040fe651a.png)

terminal output:
![Screenshot from 2021-11-09 19-14-45](https://user-images.githubusercontent.com/26383184/140962098-86543de1-b8dd-4d07-87b4-fbf2ec703a38.png)

### Operating System

Linux

### FastAPI Version

0.70.0

### Python Version

3.8.10

**Comment by oligond** [comment]
According to your Postman screenshot, you've obviously misread the [FastAPI documentation](https://fastapi.tiangolo.com/tutorial/body-multiple-params/#embed-a-single-body-parameter) -  or maybe copy-pasted the wrong portion of the example.

The (incorrect) body that you provided:

```json
{
    "name": "Foo",
    "description": "The pretender",
    "price": 42.0,
    "tax": 3.2
}
```

Instead, the properties of the item must be nested under an `item` top-level key, so please try:

```json
{
    "item": {
        "name": "Foo",
        "description": "The pretender",
        "price": 42.0,
        "tax": 3.2
    }
}
```

This is, by the way, exactly what the error response from FastAPI says:
_Hey, I'm missing a mandatory `item` property._

**Comment by birdalugureren** [comment]
I wrote items instead of item in the URL :/

**Comment by oligond** [comment]
> I wrote items instead of item in the URL :/

I don't see how this is related to the issue that you ran into.

The path `/items/9` is correct. On the contrary, if you tried to send your PUT to `/item/9`, you would end up with a 404 as your route wouldn't match the configured pattern.

Have you tried to follow my above advice and modify the request body?

**Comment by birdalugureren** [comment]
Yes you are right. It is working for embed=False. embed=True requires an item top-level key, as you stated.
I was careless, thank you.
